.eslintrc.js

```diff
@@ -103,6 +105,19 @@ module.exports = {
     'chai-expect/terminating-properties': 2,
     'curly': 2,
     'google-camelcase/google-camelcase': 2,
+
+    'import/no-unresolved': 2,
+    'import/named': 2,
+    'import/namespace': 2,
+    'import/no-useless-path-segments': 2,
+    'import/no-absolute-path': 2,
+    'import/export': 2,
+    'import/no-deprecated': 2,
+    // 'import/no-mutable-exports': 2,
+    'import/first': 2,
+    // 'import/extensions': 2,
+    // 'import/no-default-export': 2,
```

Comment:
I assume the plan is to eventually enable these rules? If so, would it make sense to uncomment the lines and set their levels to 0?

Reply:
Yes, and sure that works, will do